frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -352,6 +353,24 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
             Note: Enabling monitoring will allow any operator or workload running on this namespace
             to contribute metrics to the cluster metric set.
           </small>
+          <br />
+          <br />
+          <Alert
+            isInline
+            className="co-alert co-alert--scrollable"
+            variant="warning"
+            title="Namespace monitoring"
+          >
+            Please note that installing non RedHat operators into openshift namespaces and enabling
```

Comment:
Red Hat is two words.

```suggestion
            Please note that installing non Red Hat operators into openshift namespaces and enabling
```